package.json

```diff
@@ -15,6 +15,7 @@
     ]
   },
   "dependencies": {
+    "async": "^0.9.0",
```

Comment:
You cool using async `0.2.6` here? That is what version core ships with currently